tests/services/inventory/test_data/mock_cai_resources.dump

```diff
@@ -79,6 +79,9 @@
 {"asset_type":"iam.googleapis.com/ServiceAccount","name":"//iam.googleapis.com/projects/project2/serviceAccounts/1102","resource":{"data":{"displayName":"Compute Engine default service account","email":"1042-compute@developer.gserviceaccount.com","etag":"etag","name":"projects/project2/serviceAccounts/1042-compute@developer.gserviceaccount.com","oauth2ClientId":"1102","projectId":"project2","uniqueId":"1102"},"parent":"//cloudresourcemanager.googleapis.com/projects/1042"}}
 {"name":"//bigquery.googleapis.com/projects/project3/datasets/bq_test_ds1","asset_type":"bigquery.googleapis.com/Dataset", "resource":{"version":"v2","discovery_document_uri":"https://bigquery.googleapis.com/$discovery/rest","discovery_name":"Dataset","parent":"//cloudresourcemanager.googleapis.com/projects/1043","data":{"creationTime":"1540220016171","datasetReference":{"datasetId":"bq_test_ds1","projectId":"project3"},"location":"us","id":"project3:bq_test_ds1","kind":"bigquery#dataset","lastModifiedTime":"1540220016171"}}}
 {"name":"//bigquery.googleapis.com/projects/1042/datasets/bq_test_ds","asset_type":"bigquery.googleapis.com/Dataset", "resource":{"version":"v2","discovery_document_uri":"https://bigquery.googleapis.com/$discovery/rest","discovery_name":"Dataset","parent":"//cloudresourcemanager.googleapis.com/projects/1042","data":{"creationTime":"1540220016171","datasetReference":{"datasetId":"bq_test_ds","projectId":"project2"},"location":"us","id":"project2:bq_test_ds","kind":"bigquery#dataset","lastModifiedTime":"1540220016171"}}}
+{"name":"//bigtable.googleapis.com/projects/project2/instances/test-bigtable/clusters/test-bigtable-c1","asset_type":"bigtableadmin.googleapis.com/Cluster","resource":{"version":"v2","discovery_document_uri":"https://bigtableadmin.googleapis.com/$discovery/rest","discovery_name":"Cluster","parent":"//bigtable.googleapis.com/projects/project2/instances/test-bigtable","data":{"defaultStorageType":"SSD","location":"projects/project2/locations/us-west1-a","name":"projects/project2/instances/test-bigtable/clusters/test-bigtable-c1","state":"READY"}}}
```

Comment:
Please update the project_id based on https://github.com/forseti-security/forseti-security/blob/dev/tests/services/inventory/test_data/mock_gcp_results.py#L5979 for mock data.

Reply:
Chatted with @red2k18 offline about this. The inventory tests for CAI data are using different project Ids then the API tests and this would require making more changes.